sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>An OpenTelemetry .NET distro that exports to Azure Monitor</Description>
+    <AssemblyTitle>AzureMonitor OpenTelemetry Distro</AssemblyTitle>
+    <Version>1.0.0-alpha.1</Version>
+    <PackageTags>Azure Monitor OpenTelemetry Exporter Distro ApplicationInsights</PackageTags>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Core" />
```

Comment:
Azure.Monitor.OpenTelemetry.Exporter.csproj has indirect reference to `Azure.Core`. We don't need to add here again.

Reply:
good catch. i'll remove this now